src/styles/calendar.scss

```diff
@@ -0,0 +1,63 @@
+.calendar {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  &__date {
+    box-sizing: border-box;
+    flex-wrap: wrap;
+    display: flex;
+    width: $calendar-width;
+    gap: 1px;
+    padding: $date-padding;
+  }
+
+  &__day {
+    box-sizing: border-box;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    border: 1px solid black;
+    height: $day-height;
+    min-width: $day-width;
+    background-color: $day-color;
+    transition: 0.5s;
```

Comment:
It's better to specify to what properties you need to apply the transition.

![image](https://github.com/mate-academy/layout_calendar/assets/54860641/b350c042-f52d-4452-a77d-0e6df071010f)